src/model_repository_manager/model_repository_manager.cc

```diff
@@ -218,8 +219,7 @@ CreateAgentModelListWithLoadAction(
       std::unique_ptr<TritonRepoAgentModel> agent_model;
       if (lagent_model_list->Size() != 0) {
         lagent_model_list->Back()->Location(&artifact_type, &location);
-        const auto config_path = JoinPath({location, kModelConfigPbTxt});
-        if (!ReadTextProto(config_path, &model_config).IsOk()) {
+        if (!ReadTextProto(orignial_model_config_path, &model_config).IsOk()) {
```

Comment:
I don't understand this change. The intention is that after creating a repo agent, it may have modified the files / change model repo location, therefore the config should be reloaded from the location reported by the repo agent.

After the change the config from original model repo will always be loaded?

Reply:
Nice catch. I misunderstood the previous line where `location` can be updated.

Reply:
Fixed